### PR TITLE
Small ui tweaks(Phases widget & engagement urls)

### PR DESCRIPTION
**Pull request by djnunez-aot**
Issue : #2185 

-Change some wording & styling for phases widget & engagement urls

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/engagement/form/EngagementFormTabs/Links/PublicUrls.tsx [hunk 1]
```diff
@@ -140,10 +140,34 @@ export const PublicUrls = () => {
                             }}
                             InputProps={{
                                 startAdornment: (
-                                    <InputAdornment position="start" sx={{ height: '100%', maxHeight: '100%' }}>
```
Comment (djnunez-aot) [inline met-web/src/components/engagement/form/EngagementFormTabs/Links/PublicUrls.tsx:143]: when trying to fix the font weight for the secondary button the text color is different between both I changed it to text field to match the dashboard url input adornment

File: met-web/public/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <title>Modern Engagement Tool</title>
+        <title>EPIC.engage</title>
```
Comment (jadmsaadaot) [inline met-web/public/index.html:5]: we might need to use React Helmet and make this title dynamic according to tenant. Could you check with Saravan if it's wise

File: met-web/src/App.tsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ const App = () => {
 
         try {
             const tenant = await getTenant(basename);
-
+            document.title = tenant.name;
```
Comment (jadmsaadaot) [inline met-web/src/App.tsx:62]: better to use React Helmet

Comment (djnunez-aot) [reply met-web/src/App.tsx]: this got pushed to the wrong branch removed

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**